dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityAdminGroupRestControllerIT.java

```diff
@@ -138,6 +138,8 @@ public void postCommunityAdminGroupCreateAdminGroupSuccess() throws Exception {
                         .andDo(result -> idRef
                             .set(UUID.fromString(read(result.getResponse().getContentAsString(), "$.id")))
                         );
+        // no needs to explicitly cleanup the group created as the community comes
```

Comment:
for other reviewers please note this comment as it explains the need of the refactoring done in all the Builder about the cleanup process